src/scintillaeditor.cpp

```diff
@@ -286,6 +288,7 @@ void ScintillaEditor::applySettings()
 
 	int val = Preferences::inst()->getValue("editor/characterThreshold").toInt();
 	qsci->setAutoCompletionThreshold(val <= 0 ? 1 : val);
+	// qsci->setAutoCompletionReplaceWord(false);
```

Comment:
Maybe you were testing something options here, but I would recommend not committing new unused code like this in your Pull request.

Reply:
Sorry for that

Reply:
I have removed that comment in a new commit.